nautobot/users/tests/test_models.py

```diff
@@ -14,6 +14,19 @@ def setUp(self):
         ObjectPermission.objects.create(name="Test Permission", actions=["view", "add", "change", "delete"])
 
 
+class SavedViewTest(ModelTestCases.BaseModelTestCase):
+    model = SavedView
+
+    def setUp(self):
+        self.user = User.objects.create_user(username="Saved View test user")
+        self.sv = SavedView.objects.create(name="Saved View", owner=self.user, view="dcim:location_list")
+
+    def test_is_global_default_saved_view_is_shared_automatically(self):
+        self.sv.is_global_default = True
+        self.sv.save()
+        self.assertEqual(self.sv.is_shared, True)
+
```

Comment:
Would be good to have a test case for:
- setting one saved-view as global default unsets the is_global_default flag on an existing default for the same view
- multiple saved-views for *different* views can all have `is_global_default` set.